projects/observability/src/pages/apis/backend-detail/traces/backend-trace-list.dashboard.ts

```diff
@@ -29,6 +29,16 @@ export const backendTraceListDashboard = {
           attribute: 'name'
         }
       },
+      {
+        type: 'table-widget-column',
+        title: 'Operation',
+        width: '1',
```

Comment:
flex width support was removed (this file may be out of date) - need to use percentages or px values.